src/Summoner/Template.hs

```diff
@@ -269,6 +270,22 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
         hackageLink :: Text =
           "https://hackage.haskell.org/package/" <> repo
 
+        stackBadges :: Text = stackBadgeLts <> stackBadgeNightly
```

Comment:
I think that using `<>` here doesn't add desired line-break. Anyway, I think that some redundant code can be removed and you can just using single quasi-quoter for this variable:

```haskell
[text|
![Stackage Lts](${stackShieldLts})](${stackLinkLts})
![Stackage Nightly](${stackShieldNightly})](${stackLinkNightly})
|]
```